src/libs/PolicyUtils.ts

```diff
@@ -241,6 +241,13 @@ function getCleanedTagName(tag: string) {
     return tag?.replace(/\\{1,2}:/g, CONST.COLON);
 }
 
+/**
+ * Escape colon from tag name
+ */
+function escapeTagName(tag: string) {
+    return tag?.replaceAll(CONST.COLON, '\\:');
```

Comment:
<img width="453" alt="Screenshot 2024-05-20 at 17 20 27" src="https://github.com/Expensify/App/assets/141406735/fcfbb521-8b3e-4e64-82f2-aae4c4063ecb">

@bernhardoj I see there are many cases the BE replace `:` by `\:`

Reply:
Can you clarify what you are trying to request?

Reply:
You assume that the BE will replace `:` with `\\:` and I see there are some exceptional

Reply:
@bernhardoj If you enter the tag name is a:b the BE will return `a\:b` and the optimistic data in your change will return `a\\:b`

Reply:
Do you see any pattern in how the BE replaces the colon?

cc @amyevans